test/test_nn.py

```diff
@@ -7287,6 +7287,20 @@ def test_pickle_module_no_weights_only_warning(self):
             pickle.loads(pickle.dumps(torch.nn.Linear(10, 10)))
         self.assertEqual(len(w), 0)
 
+    def test_nllloss_1d_input_compile(self):
```

Comment:
This is not the right place to add such a test

You should add a SampleInput to the sample inputs function for nllloss here https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py?fbclid=IwY2xjawK-1IlleHRuA2FlbQIxMQBicmlkETFxd2xhenMxRXdoZG41VHFzAR7CrP-ljODBr4dDXoJQsT9-3FsMsphketNpz_kZEEaOjnxWUEpN66cs92tq_A_aem_3Y0cyFSZ1FNZGuFhSeKUWA#L8389

Running 

`python test/test_decomp.py TestDecompCUDA.test_comprehensive_nn_functional_nll_loss_cuda_float32` should then reflect the failure before your PR